### DISCUSSION TITLE
No backdrop with Bartender 4

### DISCUSSION THREAD

**Discussion by roweboy1974** [SFX-WoW/Masque#78] [Questions] [answered]
This is for Classic. I'm using Masque with Bartender and there is no backdrop. Am I missing something? I'm not using ElvUI or any other bar mod but the backdrop for the buttons is just a semi-transparent black background.

**Comment by StormFX** [comment]
The backdrop only appears when you have the button grid enabled.

Edit: Which skin, by the way?

**Comment by roweboy1974** [comment]
I have the button grid enabled. None of the skins show the backdrop but I tend to stick with "Classic"

The top bar on the left with is Bar 1. All 6 bars have the same setup (except location of course).

![Bartender](https://user-images.githubusercontent.com/22044497/65802005-6af75600-e148-11e9-8e33-3756d735577e.jpg)
![Masque](https://user-images.githubusercontent.com/22044497/65802010-6d59b000-e148-11e9-87e9-fb4a63d01a85.jpg)

**Comment by StormFX** [comment]
Hit the "Reset Skin" button.

**Comment by roweboy1974** [comment]
Tried that but that only makes the bar backdrop complete empty. 

I had the same issue with just Bartender and Masque as the only 2 addons running

**Comment by StormFX** [comment]
Reset the skin, then enable the backdrop. In your image, the backdrop color is set to black. It should be white, since the texture is an image.

**Comment by roweboy1974** [comment]
Tried that too. It's just not working for me.

**Comment by StormFX** [comment]
Try manually setting the backdrop color to white. If that doesn't work, reinstall _Masque_ and delete its saved variables files. Outside of that, I don't know what to tell you. It's working fine for me.

**Comment by roweboy1974** [comment]
Tried everything. Deleted both addons and their saved variables files and nothing. I even tried with Dominos and still nothing. I'm not the only person having the problem:

https://www.curseforge.com/wow/addons/bartender4?comment=6578

**Comment by StormFX** [comment]
Not sure what's going. on with your install.

**Comment by roweboy1974** [comment]
Well, it's working as your screenshot has it but wasn't there a time when there was a skin with the Blizzard art bar backdrop? That's what I'm looking for. As it is now it just adds a boring black semi-translucent backdrop.

**Comment by StormFX** [comment, marked as answer]
It's using the _native_ "backdrop" texture, as it's supposed to.

**Comment by roweboy1974** [comment]
Thank you. Sorry for the confusion.
